clang/test/CodeGen/attr-target-version.c

```diff
@@ -11,7 +11,7 @@ int __attribute__((target_version("fp+aes"))) fmv(void) { return 6; }
 int __attribute__((target_version("crc+ls64_v"))) fmv(void) { return 7; }
 int __attribute__((target_version("bti"))) fmv(void) { return 8; }
 int __attribute__((target_version("sme2"))) fmv(void) { return 9; }
-int __attribute__((target_version("default"))) fmv(void);
+int __attribute__((target_version("default"))) fmv(void) { return 0; }
```

Comment:
I think `fmv_one` and `fmv_too` should be defined too, to keep the original behaviour of this test unchanged, and then add a new, separate test for checking the various behaviours of `default`

Reply:
done